content/raw/chat/message_fetching.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Message Fetching spec"
+description: ""
+date: 2020-10-06T08:48:57+00:00
+lastmod: 2020-10-06T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "chat"
+weight: 100
+toc: true
+---
+
+# Message Fetching
+
+## Designs
+
+* [Fetch on Scroll](https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=7410%3A46790)
+* Message gaps
```

Comment:
Add a link to the Skeleton Loading designs here https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=8276%3A186080